integration_tests/moduleNameMapper-wrong-config/package.json

```diff
@@ -0,0 +1,7 @@
+{
+    "jest": {
+        "moduleNameMapper": {
+                "\\.(css|less)$": "module-that-dont-exist"
```

Comment:
`"module-that-dont-exist"` -> `"no-such-module"` maybe? Your name has grammar issues.